README.md

```diff
@@ -89,7 +89,7 @@ Devise.setup do |config|
   # config.expire_password_after = 3.months | true | false
 
   # Need 1 char each of: A-Z, a-z, 0-9, and a punctuation mark or symbol
-  # config.password_complexity = { digit: 1, lower: 1, symbol: 1, upper: 1 }
```

Comment:
I think I would like to see what happens if all of the complexity parameters are passed. Some explanation might be necessary to explain the side effects.

Reply:
Agreed